src/main/java/org/fcrepo/spec/testsuite/crud/HttpPatch.java

```diff
@@ -122,12 +128,49 @@ public void serverManagedPropertiesModification(final String uri) {
                                         + "4.2.4.1), the server must fail the request by responding with a 4xx range"
                                         + " status code (e.g. 409 Conflict).",
                                         "https://fcrepo.github.io/fcrepo-specification/#http-patch", ps);
+
+        //This test assumes that creating an empty container will result in RDF that contains at least one
+        // server-managed triple.
```

Comment:
Where does the Fedora specification say anything about the creation of an empty container resulting in at least one server-managed triple? In fact, last I checked, the specification has nothing to say about "server managed" triples. I believe that "server-managed triples" is a Fedora4 invention. I can assure you that Trellis allows a client to create an empty resource, resulting in....an empty resource.

Reply:
Easy:

1. try setting `ldp:contains` to some arbitrary value.
2. try setting `ldp:insertedContentRelation` on a basic container (not allowed under LDP)
3. try setting a semantically invalid statement: `<> a "literal value"`
4. try setting mutually exclusive properties on a direct container: i.e. both `ldp:hasMemberRelation` and `ldp:isMemberOfRelation` (not allowed under LDP)
5. try setting `ldp:membershipResource` to some out-of-domain resource (I'm not sure what that would even mean on an LDP server).

Surely some (or all) of those would fail on a good LDP implementation.

Reply:
Thanks.